src/pretalx/event/models/event.py

```diff
@@ -292,6 +292,7 @@ class orga_urls(EventUrls):
         plugins = '{settings}plugins'
         information = '{base}info/'
         new_information = '{base}info/new'
+        feedbacks = '{base}submissions/feedbacks/'
```

Comment:
A couple of requests:

- Could you please use the singular "feedback" over "feedbacks"?
- Could you please use `{submissions}feedback/` instead? I know the other URLs are not consistent regarding this, but it would make a URL change more readable.
- I think moving this line up to the other submission URLs would make the list a bit more cohesive and readable.